README.md

```diff
@@ -9,10 +9,10 @@ interactive graphql interface.
 Keyword Arguments
 
 - `schema` a graphene schema to handle the GraphQL queries
-- `error_handler` a function that takes two kwargs, `error` which is the specific error, and `params` which are the parameters that came into the request.
+- `error_handler` (optional, default: no-op) a function that takes two kwargs, `error` which is the specific error, and `params` which are the parameters that came into the request.
 - `result_class` (optional, default:`GraphQLResult`) a class that conforms to the same interface of `GraphQLResult`
 - `enable_graphiql` (default: True) determines whether or not the GraphiQL GUI will be available.
-
+- `context_handler` (optional, default: no-op) a fn which will return a dictionary of additional context to pass into the query being executed, ie the current user.
```

Comment:
maybe just call it `context`, and document it as either a value or a callable that can be passed. If it's callable, call it, if not just use the value.